.storybook-react/config.js

```diff
@@ -1,4 +1,5 @@
 import "!style-loader!css-loader!sass-loader!import-glob-loader!../src/styles.scss";
+import * as React from "react";
```

Comment:
is this used?

Reply:
It's needed whenever you use jsx in a file.